libs/framework/federate-component/src/server.test.tsx

```diff
@@ -18,7 +18,7 @@ const mockLoggerError = jest.fn()
 Logger.error = mockLoggerError
 
 function renderComponent(context = {}) {
-  const Component = getServerComponent(context, 'coolRemote', 'coolModule', {}, 'coolRemoteUrl')
+  const Component = getServerComponent(context, 'coolRemote', 'coolModule', {}, 'htttp://coolRemoteUrl:3000')
```

Comment:
htttp?

Reply:
haha, it's only a mock, but yeah clearly a typo, i'll fix